src/service/TransactionApi.js

```diff
@@ -112,16 +110,14 @@ class TransactionApi extends BaseEosApi {
   async getApprovedTransactions () {
     const query = `
     {
-      node(func: uid(0x18933)) {
-        trxbucket {
-          approved {
-            hash
-            uid
-            content_groups(orderasc:content_group_sequence, first:1) {
-              contents {
-                label
-                value
-              }
+      trxbucket(func: has(unapproved)) @cascade {
```

Comment:
I think this one function should be (func: has(approved)), because the function will be get all approved transaction, make sense for you?

Reply:
I fixed, I forget change that validation